app/models/upload_snapshot.rb

```diff
@@ -2,9 +2,29 @@
 class UploadSnapshot < ApplicationRecord
   belongs_to :work
   attr_writer :upload
+  has_one_attached :bitstream
```

Comment:
Why is there a bitstream here?  I thought we were just going to utilize the checksum.